system/HTTP/URI.php

```diff
@@ -177,11 +177,11 @@ public static function createURIString(
                 : ltrim($path, '/');
         }
 
-        if ($query) {
+        if ($query !== null && $query !== '') {
```

Comment:
cast to string

Reply:
Only `$query !== null` is sufficient.
Now we cannot create query string like `?0`.

Reply:
updated to ony check !==null